libbeat/docs/newdashboards.asciidoc

```diff
@@ -60,44 +74,36 @@ Options:
 ----------------------------------------------------------------------
 
 ==== dir
-The input directory with the dashboards together with its dependencies. The default value is current directory.
+The input directory that contains the dashboards and all the dependencies. The default value is the current directory.
 
 ==== url
 The Elasticsearch URL. The default value is http://localhost:9200.
 
 ==== user
-The username and password for authenticating the connection to Elasticsearch using Basic Authentication. The username and password should be separated by a colon. By default no username and password are used.
+The username and password for authenticating the connection to Elasticsearch by using Basic Authentication. The username
+and password should be separated by a colon. By default no username and password are used.
 
 ==== kibana
-The Elasticsearch index pattern where Kibana saved its configuration. The default value is `.kibana`.
+The Elasticsearch index pattern where Kibana saves its configuration. The default value is `.kibana`.
 
+[[export-dashboards]]
+=== Exporting New and Modified Beat Dashboards
 
-To import all the dashboards together with  all the dependencies (visualizations, searches and index patterns), you just
-need to run the following command in the beats repository:
-
-On Unix systems:
+To export all the dashboards for a Beat, including any new or modified dashboards and all dependencies such as
+visualizations, searches, and index patterns, you can use the python script `export_dashboards.py` from  
+https://github.com/elastic/beats/tree/master/dev-tools[dev-tools].
 
-[source,shell]
-----------------------------------------------------------------------
-../dev-tools/import_dashboards.sh -dir etc/kibana
-----------------------------------------------------------------------
+NOTE: Dashboard names must be **prefixed** with the name of the beat in order to work correctly. For example:
+Topbeat-Dashboard.
 
```

Comment:
Please also add here that before running the `export_dashboards` script the user needs to enable the python environment, that contains all the packages that are required to run the python script.

If you are running the python script for the first time, you need to create the environment by running the following commands, one by one under the `beats/dev-tools` directory:
```
 virtualenv env
 . env/bin/activate
 pip install -r requirements.txt
```
After this, the environment is created under the `env` directory and for the next runs you can just enable the environment by running the command in the `beats/dev-tools` directory:
``` 
 . env/bin/activate
```